packages/main/src/Switch.js

```diff
@@ -100,6 +95,8 @@ const metadata = {
 		/**
 		 * Sets the accessible aria name of the component.
 		 *
+		 * <b>Note</b>: We recommend that you set an accessibleNameRef to an external label or at least an <code>accessibleName</code>.
```

Comment:
Sorry, let's make it clearer:
pointing to an external label

Reply:
Done